lib/task_bunny/config.ex

```diff
@@ -185,4 +185,18 @@ defmodule TaskBunny.Config do
       _ -> []
     end
   end
+
+  @doc """
+  Returns the list of failure backends.
+
+  It returns [TaskBunny.FailureBackend.Logger] by default.
```

Comment:
I think that if you put `TaskBunny.FailureBackend.Logger` between **backticks** that ExDoc will pick up on the module and make it linkable.

Ref: [Auto-Linking](https://github.com/elixir-lang/ex_doc#auto-linking)